AUTHORS

```diff
@@ -4,7 +4,7 @@ those who explicitly didn't want to be mentioned. People with a * next
 to their names are not found in the metadata of the git history. This
 file is generated automatically by running `./bin/authors_update.py`.
 
-There are a total of 1127 authors.
+There are a total of 1137 authors.
```

Comment:
This change also needs to be reverted.

When looking at the PR you can click "files changed" at the top to see what is actually being changed.

Reply:
So there should be 1126 authors now, since I removed myself, right?

Reply:
The number will be updated at the time of the next release. For now it is best just to leave it unchanged.

The reason for not changing things like this is that if any two PRs changes the same line of the same file then there will be merge conflicts. If every PR tries to change this line then every PR will have merge conflicts.